src/mrpro/data/_KNoise.py

```diff
@@ -0,0 +1,61 @@
+"""MR noise measurements class."""
+
+# Copyright 2023 Physikalisch-Technische Bundesanstalt
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+from __future__ import annotations
+
+import dataclasses
+from pathlib import Path
+
+import ismrmrd
+import torch
+from einops import rearrange
+
+from mrpro.data.enums import AcqFlags
+
+
+@dataclasses.dataclass(slots=True, frozen=True)
+class KNoise:
```

Comment:
We could consider making the Flags a parameter in KData.
And KNoise inherit from KData with changed default flag selection ( keep only noise)

Reply:
I would prefer to keep it separate rather than KNoise having a header and trajectory which are invalid.